tests/integration/semantics/src/test/java/org/apache/pulsar/tests/integration/functions/runtime/PulsarFunctionsRuntimeTest.java

```diff
@@ -57,7 +58,7 @@ public PulsarFunctionsRuntimeTest(ContainerFactory containerFactory) {
 
     @Test(dataProvider = "FunctionRuntimes")
     public void testExclamationFunction(Runtime runtime) throws Exception {
-        if (ContainerFactory.THREAD == containerFactory && Runtime.PYTHON == runtime) {
+        if (functionProcessingMode == FunctionProcessingMode.THREAD && runtime == Runtime.PYTHON) {
```

Comment:
why do you change it here?

Reply:
lvalue are suppose to be variables by convention

Reply:
sure. if you would like to introduce this coding convention, I would recommend doing it in a separate PR, and describe that clearly in the PR's description. or introduce checkstyle to enforce this coding convention.

because this change here doesn't belong to this PR, it is unrelated to any other changes in PR.
when people look back to the git history, it is really hard to know why did you change this line.